packages/Teams/js/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@microsoft/bot-components-teams",
+  "author": "Microsoft Corp.",
+  "description": "Rule system for the Microsoft BotBuilder adaptive dialog system, with integration specific to Microsoft Teams.",
+  "version": "1.0.0",
+  "license": "MIT",
+  "keywords": [
+    "botbuilder",
```

Comment:
We'll need to update the keywords so that it appears in Package Manager. At the very least, I think that entails having the msbot-component keyword. There are other ones that I believe Andrew can provide and give guidance on what to use.

Reply:
@clearab can you verify what tags we need for components to show up in package manager?